src/providers/oracle/qgsoraclesourceselect.h

```diff
@@ -134,7 +134,7 @@ class QgsOracleSourceSelect : public QDialog, private Ui::QgsDbSourceSelectBase
     //!Sets a new regular expression to the model
     void setSearchExpression( const QString& regexp );
 
-    void on_buttonBox_helpRequested() { QgsContextHelp::run( metaObject()->className() ); }
+    void on_buttonBox_helpRequested() { QgsHelp::openHelp( QStringLiteral( "working_with_vector/supported_data.html#id2" ) ); }
```

Comment:
Ok this is what I understood. But what changes does this require from the way we currently write docs? What should I be careful to not change the next time I update the doc (note that this chapter particularly is supposed to be reorganized in the future)? 
> When the help button is clicked, the browser will be opened with the url http://docs.qgis.org/2.14/en/docs/user_manual/working_with_vector/supported_data.html#id2

shouldn't it rather be this one http://docs.qgis.org/2.14/en/docs/user_manual/working_with_vector/supported_data.html#oracle-spatial-layers (the permanent link and not the relative one - which is btw #id25 and not #id2)?